### ISSUE TITLE
Right click with egg or experience bottle

### ISSUE THREAD

**Issue by Chopnex** [Matocolotoe/Skript-1.8#89]
### Skript Version

[17:48:32 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[17:48:32 INFO]: [Skript] Skript's documentation can be found here: https://docs.skriptlang.org/
[17:48:32 INFO]: [Skript] Skript's tutorials can be found here: https://docs.skriptlang.org/tutorials
[17:48:32 INFO]: [Skript] Server Version: git-PaperSpigot-94ac179e (MC: 1.8.8)
[17:48:32 INFO]: [Skript] Skript Version: 2.6.5-for-1.8
[17:48:32 INFO]: [Skript] Installed Skript Addons: 
[17:48:32 INFO]: [Skript]  - skript-placeholders v1.5.2 (https://github.com/APickledWalrus/skript-placeholders)
[17:48:32 INFO]: [Skript]  - skript-gui v1.3 (https://github.com/APickledWalrus/skript-gui)
[17:48:32 INFO]: [Skript]  - skDragon v0.17
[17:48:32 INFO]: [Skript]  - skript-reflect v2.3 (https://github.com/TPGamesNL/skript-reflect)
[17:48:32 INFO]: [Skript]  - skRayFall v1.9.25 (https://sk.rayfall.net/)
[17:48:32 INFO]: [Skript]  - SkQuery v4.1.6
[17:48:32 INFO]: [Skript]  - SkBee v1.10.3 (https://github.com/ShaneBeee/SkBee)
[17:48:32 INFO]: [Skript]  - WolvSK v1.7.2
[17:48:32 INFO]: [Skript]  - LabyModSK v1.1.0
[17:48:32 INFO]: [Skript]  - MythicSkriptAddon v0.97a-SNAPSHOT
[17:48:32 INFO]: [Skript] Installed dependencies: 
[17:48:32 INFO]: [Skript]  - Vault v1.7.3-CMI
[17:48:32 INFO]: [Skript]  - WorldGuard v6.1.3-SNAPSHOT;c904242

### Bug Description

When using:

aliases:
	splashitem = egg, experience bottle
	
on right click with splashitem:
	cancel event

The event is NOT cancelled when the player is facing a entity such as a NPC.

### Expected Behavior

The event should cancel no matter what the target entity is

### Steps to Reproduce

See above

### Errors or Screenshots

```
Replace this text with your paste link or errors/screenshots/videos.
```

### Agreement

- [X] I have read the guidelines above and confirm I am following them with this report.

**Comment by Chopnex** [comment]
It is probably the same with snowballs and other throwables

**Comment by Chopnex** [comment]
Workaround for any wondering, you need to cancel the on shoot event for the projectiles such as egg and experience bottle etc.

**Comment by Matocolotoe** [comment]
> Workaround for any wondering, you need to cancel the on shoot event for the projectiles such as egg and experience bottle etc.

That's probably the expected behavior, since entity interactions do not count as right clicking as intended in Skript.

**Comment by Chopnex** [comment]
Yes, its probably a minecraft bug anyways. The problem is just that the item is thrown when the target entity is set. I just choose not to use the throwables on the server

**Comment by Matocolotoe** [comment]
I've had several issues with projectiles as items and entities as well, so I guess it is. You could probably try cancelling PlayerInteractEntityEvent or PlayerInteractAtEntityEvent with skript-reflect, as a sort of workaround.
